src/client/reducers/reducer_searchBy.js

```diff
@@ -0,0 +1,22 @@
+import C from '../actions/constants';
+
+const searchBy = [
+    { title: 'Title', type: 'title', active: true },
+    { title: 'Director', type: 'director', active: false },
+];
+
+export default function (state = searchBy, action) {
```

Comment:
@vitkuz what if we create only one search reducer which default state  will look smth like this
```
query: ''
filter: director || title || undefined
```

that will allow you to do a search and won't contain additional/excessive info

Reply:
@algbeta I removed it.